cardano-api/cardano-api.cabal

```diff
@@ -171,9 +172,11 @@ test-suite cardano-api-test
                       , text
                       , time
 
-  other-modules:        Test.Cardano.Api.Crypto
+  other-modules:        Gen.Genesis
```

Comment:
Let's put these generators in the existing `Gen` module.

Reply:
Moved to `Test.Cardano.Api.Gen`